src/test/account.test.js

```diff
@@ -222,7 +222,7 @@ describe("Get Account by ID", () => {
     const token = await databaseConnection();
 
     const response = await api
-      .get("/api/v1/account/613811dbae8adc538c0c2b24")
+      .get("/api/v1/account/61414ff50aaf6d3f34f1abe3")
       .set("Authorization", `Bearer ${token}`);
 
     expect(response.statusCode).toEqual(200);
```

Comment:
Why in the test you said that should return a 400 status, but you are expecting 200? @raiandexter0607

Reply:
are two test differents

Reply:
are two different test.